docs/product-user-guide/src/main/asciidoc/test-scenarios-con.adoc

```diff
@@ -1,11 +1,8 @@
-[id='test_scenarios_con']
-= Test Scenarios
+[id='test-scenarios-con']
+= Test scenarios
 
-Test Scenarios in {PRODUCT} enable you to validate the functionality of rules, models, and events before deploying them into production. A test scenario uses data for conditions that resemble an instance of your fact or project model. This data is matched against a given set of rules and if the expected results match the actual results, the test is successful. If the expected results do not match the actual results, then the test fails.
+Test scenarios in {PRODUCT} enable you to validate the functionality of business rules and business rule data before deploying them into production. With a test scenario, you use data from your project to set given conditions and expected results based on one or more defined business rules. When you run the scenario, the expected results and actual results of the rule instance are compared. If the expected results match the actual results, the test is successful. If the expected results do not match the actual results, then the test fails.
 
-After you run all test scenarios, the status of the scenarios is reported in a *Messages* panel.
+Test scenarios can be executed one at the time or as a group. The group execution contains all the scenarios from one package. Test scenarios are independent, so one scenario cannot affect or modify the other. You can run test scenarios at any time during project development in {CENTRAL}. You do not have to compile or deploy your decision service to run test scenarios.
 
-.Speed test scenario screen
-image::speed-test-scenario.png[Speed test Scenario screen]
-
-Test scenarios can be executed one at the time or as a group. The group execution contains all the scenarios from one package. Test scenarios are independent, so that one scenario cannot affect or modify the other.
+All data objects related to a test scenario must be in the same project package as the test scenario. Assets in the same package are imported by default. After the necessary data objects and the test scenario are created, you can use the *Data Objects* tab of the test scenarios designer to verify that all required data objects are listed or to import other existing data objects by adding a *New item*.
```

Comment:
...objects by adding a *New item*. I dont' think this should be capped and bolded. You are talking generally about adding an item, not telling the user to click New item. Similar about *Person* in intro to chap 2. Dunno? Actually, this happens throughout, so I guess it's OK.  Later Enter a unique *Data Object* name. I would day Enter a unique data object name or In the *Data Object* box enter a name...

Reply:
Yeah, you and I have discussed this principle a couple of times before. It's just an economy to mention an item and point the user to the UI at the same time. IBM says not to do this, but this is just one of a few things I deliberately flout as an economy. Gets very clunky very fast otherwise, or underinforms the user about the UI. I'm open to thoughts, but there are pros and cons both ways.